Sources/StreamChatUI/UIConfig+Images.swift

```diff
@@ -15,6 +15,7 @@ public extension _UIConfig {
         public var back: UIImage = UIImage(named: "icn_back", in: .streamChatUI)!
         public var onlyVisibleToCurrentUser = UIImage(named: "eye", in: .streamChatUI)!
         public var more = UIImage(named: "icn_more", in: .streamChatUI)!
+        public var share = UIImage(named: "share", in: .streamChatUI)!
```

Comment:
I'd say we should use the default system icon for this and only when it's not available use the custom one.